rune/libenclave/internal/runtime/pal/skeleton/sgxsign.c

```diff
@@ -519,17 +525,20 @@ int main(int argc, char **argv)
 	RSA *sign_key;
 	bool enclave_debug = true;
 	struct metadata meta_data;
-	char *const short_options = "ps:x:a:";
+	char *const short_options = "ps:x:a:nm:";
 	struct option long_options[] = {
 		{"product", no_argument, NULL, 'p'},
 		{"mmap-size", required_argument, NULL, 's'},
 		{"xfrm", required_argument, NULL, 'x'},
 		{"attrs", required_argument, NULL, 'a'},
+		{"null_dereference_protection", no_argument, NULL, 'n'},
+		{"mmap_min_addr", required_argument, NULL, 'm'},
 		{0, 0, 0, 0}
 	};
 
 	program = argv[0];
 	memset(&meta_data, 0, sizeof(struct metadata));
+	meta_data.mmap_min_addr = mmap_min_addr;
```

Comment:
Why mmap_min_addr is required to be specified explicitly? I understand that this was an issue because mmap_min_addr is guessed during signing and thus we ware assuming the signing machine is the running machine. However, introducing metadata area intends to overcome this problem. So I hope to see the assumption mentioned above is removed in this pr.